recipes/leveldb/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+
+### Build and check
+
+make
+make check
+
+
+### Install manually ( no make install :) )
+
+# Ensure there is somewhere to put this stuff
+mkdir -p $PREFIX/include
+mkdir -p $PREFIX/lib
```

Comment:
Shouldn't be a problem, but figured I would make sure these directories are created if not already there.